lib/player.rb

```diff
@@ -32,11 +35,16 @@ def highest_word_score
     end
 
     def won?
-      if @total > 100
-        true
-      else
-        false
+      total_score > 100
+    end
+
+    def draw_tiles(tile_bag)
+      @bag = tile_bag
```

Comment:
What is the reason for having `@bag` as an instance variable rather than `bag` as a local variable?